backend/src/Services/Logging/LoggerImpl.ts

```diff
@@ -8,33 +8,42 @@ export class LoggerImpl {
     }
 
     debug(context?:ILoggingContext,  message?: any, ...optionalParams: any[]):void{
-        this.log(context, "debug", message, ...optionalParams);
+        this.log(context, "", message, ...optionalParams);
     }
 
     info(context?:ILoggingContext,  message?: any, ...optionalParams: any[]):void{
-        this.log(context, "info", message, ...optionalParams);
+        this.log(context, "", message, ...optionalParams);
     }
 
     warn(context?:ILoggingContext,  message?: any, ...optionalParams: any[]):void{
-        this.log(context, "WARN", message, ...optionalParams);
+        this.log(context, "WARN ", message, ...optionalParams);
     }
 
     error(context?:ILoggingContext,  message?: any, ...optionalParams: any[]):void{
+        //TODO - put more structure to the data shared in request and spit it all out here
         this.log(context, "ERROR", message, ...optionalParams);
     }
 
     log(context?:ILoggingContext, levelStr?:string,  message?: any, ...optionalParams: any[]):void { 
-        var msg = new Date(Date.now()).toISOString();
+        var msg = "";
+        var lvlStr = "";
+        var ctxStr = "";
+        var prefix = "";
         if (context!= null){
-            if (context.contextId == null || context.contextId == undefined){
-                context.contextId = randomUUID();
+            ctxStr = "  ctx:" + context.contextId;
```

Comment:
Good idea adding the context id at the end! Looks like it will be useful for filtering